src/main/java/kg/jarkyn/cobspecserver/Server.java

```diff
@@ -4,11 +4,15 @@
 import kg.jarkyn.cobspecserver.sockets.ClientSocket;
 import kg.jarkyn.cobspecserver.sockets.ListenerSocket;
 
+import java.util.concurrent.ExecutorService;
+
 public class Server {
+    private ExecutorService executor;
     private ListenerSocket listener;
     private ResponseController responseController;
 
-    public Server(ListenerSocket listener, ResponseController responseController) {
+    public Server(ExecutorService executor, ListenerSocket listener, ResponseController responseController) {
```

Comment:
would be good to talk about it more! thank you, and yes, not sure I actually have enough time for this before today.